doc/PythonSupport.md

```diff
@@ -0,0 +1,64 @@
+# Mystikos Support and Limitations for Python
+**NOTE**: this is a living document that is continually updated.
+
+## Verified/Supported python versions
+As stated in our [Getting started with Python](user-getting-started-docker-python.md) document, we recommend building your containerized python application using python 3.8.11 and 3.9.7. Note that if possible, try pin your python to a specific version like 3.8.11 instead of 3.8. Even when this document was written, there are newer python release than 3.8.11 or 3.9.7. We will do our best to stablize and support mystikos on newer versions.
+
+### Install position independent python executable
+If you run `apt install python3.8` when building a Ubuntu-based docker image, the installed python3.8 (or python3.9) executable is not a position independent executable (PIE). `myst exec <cmd>` requires the cmd to be a PIE. To overcome this, either install python using miniconda (reference our [python_webserver](../solutions/python_webserver/Dockerfile) example), or use a Debian base image instead. For example,
```

Comment:
Please break long longs into shorter ones (~80 characters per line).

The 3rd option is to build cpython from source following instructions at https://github.com/python/cpython#build-instructions

Reply:
Done.